src/main/java/com/microsoft/aad/msal4j/AadInstanceDiscoveryProvider.java

```diff
@@ -180,9 +205,12 @@ private static IHttpResponse httpRequest(String requestUrl, Map<String, String>
 
     private static String discoverRegion(MsalRequest msalRequest, ServiceBundle serviceBundle) {
```

Comment:
msalRequest does not seem to used

Reply:
It's referenced in the try/catch block of that method where it makes the call to the IMDS endpoint. It's needed in the `httpRequest` method that makes the actual call.